test/unit/rules/no-this-in-template-only-components-test.js

```diff
@@ -16,6 +16,24 @@ generateRuleTests({
         filePath: 'app/templates/route-template.hbs',
       },
     },
+    {
+      template: '{{this.name}}',
+      meta: {
+        filePath: 'app/templates/route-template.gts',
+      },
+    },
+    {
+      template: '{{this.data}}',
+      meta: {
+        filePath: 'addon/templates/components/foo-bar.gjs',
+      },
+    },
+    {
+      template: '{{this.bar}}',
```

Comment:
these aren't gjs 🙈

Reply:
Ohh, @NullVoxPopuli you are right! Changes pushed